conf/cassandra-env.sh

```diff
@@ -199,6 +199,44 @@ if [ "x$CASSANDRA_HEAPDUMP_DIR" = "x" ]; then
 fi
 JVM_OPTS="$JVM_OPTS -XX:HeapDumpPath=$CASSANDRA_HEAPDUMP_DIR/cassandra-`date +%s`-pid$$.hprof"
 
+# Cassandra heap dump files management options:
+# if not set externally, this script assigns defauls:
+#  - enable heap dump files clean up
+#  - keeping 2 newest files
+#  - keeping 1 oldest file to help identify first OOM incident
+if [ "$CASSANDRA_HEAPDUMP_CLEAN" = "" ]; then
```

Comment:
what is about not introducing a separate CASSANDRA_HEAPDUMP_CLEAN option here and just use a negative value for CASSANDRA_HEAPDUMP_KEEP_NEWEST_N_FILES to disable this feature?

Reply:
agree. fixed